lib/wallet/cardano-wallet.cabal

```diff
@@ -806,16 +806,26 @@ common integration-common
     , unliftio
     , with-utf8
 
-executable integration-exe
+executable babbage-integration-exe
   import:  integration-common
   main-is: babbage-integration-tests.hs
 
-test-suite integration
+executable conway-integration-exe
+  import:  integration-common
+  main-is: conway-integration-tests.hs
+
+test-suite babbage-integration
   import:             integration-common
   type:               exitcode-stdio-1.0
   main-is:            babbage-integration-tests.hs
   build-tool-depends: cardano-wallet:cardano-wallet
 
+test-suite conway-integration
+  import:             integration-common
+  type:               exitcode-stdio-1.0
+  main-is:            conway-integration-tests.hs
+  build-tool-depends: cardano-wallet:cardano-wallet
+  
```

Comment:
Traditionally we have had a single test suite, at times with some era-dependent expectations. Why not do it this way? E.g. by marking all tests `pendingOnConway` for now

Reply:
One current oddity I think is `conway-integration` and `babbage-integration` both having `BabbageEra` as default era...

Reply:
Yes we need to discuss this choice.